locales/bg_BG/LC_MESSAGES/duckduckgo.po

```diff
@@ -2533,6 +2534,7 @@ msgctxt "mobile homepage banner"
 msgid "Open %sMenu%s"
 msgstr ""
 
+#, fuzzy,translated-trademark
 msgid "Open %sSettings > Safari > Search Engine%s and select DuckDuckGo!"
 msgstr "Отворете %sНастройки > Safari > Търсачка%s и изберете!"
```

Comment:
These are kinda weird, it looks like "DuckDuckGo" is missing there. Like, the sentences look like they've had a word chopped off. Hope you don't mind if I jump in and fix it.